### PR TITLE
[Actions] Add spell checker

### PR DESCRIPTION
**Pull request by gichan-jang**
- Add spell checker github actions.
 - Fix all typos.
 - Need to check typo in flatbuf and protobuf schema. (Now, skip the check for this)

Signed-off-by: gichan2-jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [* ]Passed [ ]Failed [ ]Skipped
2. Run test: [* ]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by jaeyun-jung — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ The TSC voting members are:
 ## TSC Meeting
 
 
-A TSC meeting is required to be publically announced (at least 10 days before the meeting) and publically accessible. A chairperson or its deputy, who is designated by a chairperson or the former chairperson, may announce and hold a TSC meeting. A deputy designated by a former resigning chairperson should hold a TSC meeting to elect a new chairperson and is relieved automatically by electing a new chairperson.
+A TSC meeting is required to be Publicly announced (at least 10 days before the meeting) and Publicly accessible. A chairperson or its deputy, who is designated by a chairperson or the former chairperson, may announce and hold a TSC meeting. A deputy designated by a former resigning chairperson should hold a TSC meeting to elect a new chairperson and is relieved automatically by electing a new chairperson.
```
Comment (jaeyun-jung) [inline CONTRIBUTING.md:34]: Publicly > publicly

File: ext/nnstreamer/tensor_decoder/tensordec-pose.c [hunk 2]
```diff
@@ -197,7 +197,7 @@ typedef struct pose_metadata_s pose_metadata_t;
 static singleLineSprite_t singleLineSprite;
 
 /**
- * @brief Data structure for boundig box info.
+ * @brief Data structure for bounding box info.
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_decoder/tensordec-pose.c:200]: bounding box info > pose estimation info  ?

File: ext/nnstreamer/tensor_decoder/tensordec-pose.c [hunk 3]
```diff
@@ -535,7 +535,7 @@ is_value_within(int value, int lower_b, int upper_b)
 /**
  * @brief Fill in pixel with PIXEL_VALUE at x,y position. Make thicker (x+1, y+1)
  * @param[out] out_info The output buffer (RGBA plain)
- * @param[in] bdata The bouding-box internal data.
+ * @param[in] bdata The bounding-box internal data.
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_decoder/tensordec-pose.c:538]: Fix param name: data The pose-estimation internal data.

File: ext/nnstreamer/tensor_decoder/tensordec-pose.c [hunk 4]
```diff
@@ -559,7 +559,7 @@ setpixel (uint32_t * frame, pose_data * data, int x, int y)
 /**
  * @brief Draw line with dot at the end of line
  * @param[out] out_info The output buffer (RGBA plain)
- * @param[in] bdata The bouding-box internal data.
+ * @param[in] bdata The bounding-box internal data.
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_decoder/tensordec-pose.c:562]: same, data The pose-estimation internal data.

File: ext/nnstreamer/tensor_decoder/tensordec-pose.c [hunk 5]
```diff
@@ -626,9 +626,9 @@ draw_line_with_dot (uint32_t * frame, pose_data * data, int x1, int y1, int x2,
 }
 
 /**
- * @brief Draw lable with the given results (pose) to the output buffer
+ * @brief Draw label with the given results (pose) to the output buffer
  * @param[out] out_info The output buffer (RGBA plain)
- * @param[in] bdata The bouding-box internal data.
+ * @param[in] bdata The bounding-box internal data.
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_decoder/tensordec-pose.c:631]: same, data The pose-estimation internal data.

File: ext/nnstreamer/tensor_decoder/tensordec-pose.c [hunk 6]
```diff
@@ -674,7 +674,7 @@ draw_label (uint32_t * frame, pose_data * data, pose * xydata)
 /**
  * @brief Draw with the given results (pose) to the output buffer
  * @param[out] out_info The output buffer (RGBA plain)
- * @param[in] bdata The bouding-box internal data.
+ * @param[in] bdata The bounding-box internal data.
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_decoder/tensordec-pose.c:677]: same, same, data The pose-estimation internal data.

File: ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c [hunk 7]
```diff
@@ -315,7 +315,7 @@ nnfw_close (const GstTensorFilterProperties * prop, void **private_data)
  * @brief Convert from nnfw type to gst tensor type
  * @param[in] nnfw_type type given in nnfw format
  * @param[out] type container to receive type in gst tensor format
- * @return 0 on sucess, errno on error
+ * @return 0 on success, errno on error
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c:318]: suggestion: 0 on success, negative errno on error

File: ext/nnstreamer/tensor_filter/tensor_filter_tensorrt.cc [hunk 8]
```diff
@@ -214,7 +214,7 @@ tensorrt_subplugin::configure_instance (const GstTensorFilterProperties *prop)
 void
 tensorrt_subplugin::invoke (const GstTensorMemory *input, GstTensorMemory *output)
 {
-  /* If internal _inputBuffer is nullptr, tne allocate GPU memory */
+  /* If internal _inputBuffer is nullptr, tnen allocate GPU memory */
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_filter/tensor_filter_tensorrt.cc:217]: then

File: gst/edge/edge_elements.c [hunk 9]
```diff
@@ -39,6 +39,6 @@ plugin_init (GstPlugin * plugin)
 #endif
 
 GST_PLUGIN_DEFINE (GST_VERSION_MAJOR, GST_VERSION_MINOR, edge,
-    "A collcetion of GStreamer plugins to support Edge",
+    "Collcetions of GStreamer plugins to support Edge",
```
Comment (jaeyun-jung) [inline gst/edge/edge_elements.c:42]: A collection

File: gst/mqtt/mqttsrc.c [hunk 10]
```diff
@@ -748,7 +748,7 @@ gst_mqtt_src_create (GstBaseSrc * basesrc, guint64 offset, guint size,
       GstClockTime ulatency = GST_CLOCK_TIME_NONE;
       GstClock *clock;
 
-      /** This buffer is comming from the past. Drop it */
+      /** This buffer is coming from the past. Drop it */
```
Comment (jaeyun-jung) [inline gst/mqtt/mqttsrc.c:751]: This buffer is coming from the past. Drop it.

File: gst/nnstreamer/elements/gsttensor_demux.c [hunk 11]
```diff
@@ -227,8 +227,8 @@ gst_tensor_demux_dispose (GObject * object)
 
 /**
  * @brief Parse caps and configure tensors info.
- * @param tensor_demux GstTensorDemux Ojbect
- * @param caps incomming capablity
+ * @param tensor_demux GstTensorDemux Object
```
Comment (jaeyun-jung) [inline gst/nnstreamer/elements/gsttensor_demux.c:230]: Object > object

File: gst/nnstreamer/elements/gsttensor_if.c [hunk 12]
```diff
@@ -671,8 +671,8 @@ gst_tensor_if_install_properties (GObjectClass * gobject_class)
 
 /**
  * @brief Parse caps and configure tensors info.
- * @param tensor_if GstTensorIf Ojbect
- * @param caps incomming capablity
+ * @param tensor_if GstTensorIf Object
```
Comment (jaeyun-jung) [inline gst/nnstreamer/elements/gsttensor_if.c:674]: Object > object

File: gst/nnstreamer/elements/gsttensor_split.c [hunk 13]
```diff
@@ -212,8 +212,8 @@ gst_tensor_split_finalize (GObject * object)
 
 /**
  * @brief Set Caps in pad.
- * @param split GstTensorSplit Ojbect
- * @param caps incomming capablity
+ * @param split GstTensorSplit Object
```
Comment (jaeyun-jung) [inline gst/nnstreamer/elements/gsttensor_split.c:215]: Object > object

**Comment by gichan-jang** [comment]
@jaeyun-jung , Updated all comments.

File: gst/nnstreamer/elements/gsttensor_split.c [hunk 14]
```diff
@@ -404,14 +404,14 @@ gst_tensor_split_combine_flows (GstTensorSplit * split,
 }
 
 /**
- * @brief Make Splited Tensor
+ * @brief Make splitted Tensor
```
Comment (jaeyun-jung) [inline gst/nnstreamer/elements/gsttensor_split.c:407]: splitted Tensor > splitted tensor

File: gst/nnstreamer/elements/gsttensor_split.c [hunk 15]
```diff
@@ -404,14 +404,14 @@ gst_tensor_split_combine_flows (GstTensorSplit * split,
...
  * @param split TensorSplit Object
```
Comment (jaeyun-jung) [inline gst/nnstreamer/elements/gsttensor_split.c:408]: TensorSplit Object > TensorSplit object

File: tests/nnstreamer_filter_mxnet/Predictor.hh [hunk 16]
```diff
@@ -155,7 +155,7 @@ Predictor::Predictor (const std::string &model_json_file, const std::string &mod
   }
   // Load the model
   LoadModel (model_json_file);
-  // Initilize the parameters
+  // Initialize the parameters
```
Comment (jaeyun-jung) [inline tests/nnstreamer_filter_mxnet/Predictor.hh:158]: suggestion : line-comment rule
I suggest fixing line-comment rule of this file (// -> /* */)

Comment (gichan-jang) [reply tests/nnstreamer_filter_mxnet/Predictor.hh]: Good, Updated!

**Review by niley7464 — APPROVED** [review]
There were lots of tyops we missed 😅
Now, it seems that reviews related to typos will be resolved automatically.
Thank you for your hard work.

**Review by myungjoo — APPROVED** [review]

**Pull request merged**